Dockerfile

```diff
@@ -1,11 +1,35 @@
+ARG HOST_UID=1000
```

Comment:
Same.
Seams to be part of the docker PR ( #203 )

Reply:
Yes, same here. I also rebased this one.